File: c/negativos_matriz.c
#include <stdio.h>

int main(){
    int m, n;

    printf("Qual a quantidade de linhas da matriz? ");
    scanf("%d", &m);
    printf("Qual a quantidade de colunas da matriz? ");
    scanf("%d", &n);

    int matriz[m][n];

    for (int i=0; i<m; i++) {
        for (int j=0; j<n; j++) {
			printf("Elemento [%d,%d]: ", i, j);
            scanf("%d", &matriz[i][j]);
        }
    }
	
    printf("VALORES NEGATIVOS:\n");

    for (int i=0; i<m; i++) {
        for (int j=0; j<n; j++) {
            if (matriz[i][j] < 0) {
                printf("%d\n", matriz[i][j]);
            }
        }
    }

    return 0;
}

File: python/negativos_matriz.py
m:int ; n: int

m = int(input("Qual a quantidade de linhas da matriz? "))
n = int(input("Qual a quantidade de colunas da matriz? "))

matriz: [[int]] = [[0 for x in range(n)] for x in range(m)]

for i in range(m):
	for j in range(n):
		matriz[i][j] = int(input(f"Elemento [{i},{j}]: "))

print("VALORES NEGATIVOS:")

for i in range(m):
	for j in range(n):
		if matriz[i][j] < 0:
			print(matriz[i][j])